### PR TITLE
Fix some SonarQube issues

### PR DESCRIPTION
**Pull request by Tutul-**
[WinePrefix] rename to comply with Java format and log missing error
[AppPanel] rename to comply with Java format
           + Change HTMLAnchorElementImpl to use Apache instead of sun
           + fix some conversion on log missing error

### CODE REVIEW

**Comment by qparis** [comment]
Does the install bouton still work?

File: src/main/java/com/playonlinux/ui/impl/javafx/mainwindow/apps/AppPanel.java [hunk 1]
```diff
@@ -18,33 +18,28 @@
...
-import javafx.event.EventHandler;
 import javafx.scene.control.ScrollPane;
-import javafx.scene.layout.FlowPane;
 import javafx.scene.layout.HBox;
 import javafx.scene.layout.VBox;
-import javafx.scene.web.WebEvent;
 import javafx.scene.web.WebView;
+
 import org.apache.log4j.Logger;
+import org.python.apache.html.dom.HTMLAnchorElementImpl;
```
Comment (qparis) [inline src/main/java/com/playonlinux/ui/impl/javafx/mainwindow/apps/AppPanel.java]: Why is it needed?

Comment (Tutul-) [inline src/main/java/com/playonlinux/ui/impl/javafx/mainwindow/apps/AppPanel.java]: use a sun import is deprecated

File: pom.xml [hunk 2]
```diff
@@ -118,8 +118,6 @@
         </dependency>
     </dependencies>
 
-
-
     <build>
         <resources>
             <resource>
```
Comment (Tutul-) [inline pom.xml]: Sorry for the change, test some dependencies for the import but delete all my test

**Pull request closed without merging**